docker/web/Dockerfile

```diff
@@ -1,5 +1,29 @@
 FROM python:3.8
 
+# build config
+ARG HTCONDOR_VERSION=8.9
+
+# environment setup
+ENV DEBIAN_FRONTEND=noninteractive \
+    LC_ALL=en_US.UTF-8 \
+    LANG=en_US.UTF-8 \
+    LANGUAGE=en_US.UTF-8
+
+# install utils and dependencies
+# install HTCondor version specified in config
+RUN : \
+    && apt-get update \
+    && apt-get -y install --no-install-recommends vim less git gnupg wget ca-certificates locales \
```

Comment:
`vim`?  Really?

Reply:
:man_shrugging: it seemed like a good idea at the time when I needed to do some debugging, but I suppose that the image will always be run as root, so extra tools can be added at debugging time.